Cargo.toml

```diff
@@ -1,9 +1,22 @@
 [package]
 name = "saito-rust"
 version = "0.1.0"
-authors = ["Clayton Rabenda <clay@saito.tech>"]
+authors = ["Clayton Rabenda <clay@saito.tech>", "Stephe Peterkins <stephen@saito.tech>"]
```

Comment:
LOL, well, I didn't think I'd have to review this line but you mis-spelled your own name :P

Reply:
LMAO fail